test/scripts/kind_dual_conversion.sh

```diff
@@ -0,0 +1,150 @@
+    # OVN has already the SCTP feature enable, we just need to append the new one
+    docker exec $n sed -i -e "s#.*feature-gates.*#&\,IPv6DualStack=true#" ${API_CONFIG_FILE}
+
+    # kube-controller-manager /etc/kubernetes/manifests/kube-controller-manager.yaml
+    docker exec $n cp ${CM_CONFIG_FILE} /kind/${CM_CONFIG_FILE}.bak
+    # append second service cidr --service-cluster-ip-range=<IPv4 CIDR>,<IPv6 CIDR>
+    docker exec $n sed -i -e "s#.*service-cluster-ip-range.*#&\,${SECONDARY_SERVICE_SUBNET}#" ${CM_CONFIG_FILE}
+    # append second cluster cidr --cluster-cidr=<IPv4 CIDR>,<IPv6 CIDR>
+    docker exec $n sed -i -e "s#.*cluster-cidr.*#&\,${SECONDARY_CLUSTER_SUBNET}#" ${CM_CONFIG_FILE}
+    # configure feature gate --feature-gates=IPv6DualStack=true
+    # docker exec $n sed -i '/service-cluster-ip-range/i\    - --feature-gates=IPv6DualStack=true' ${CM_CONFIG_FILE}
+    # OVN has already the SCTP feature enable, we just need to append the new one
+    docker exec $n sed -i -e "s#.*feature-gates.*#&\,IPv6DualStack=true#" ${CM_CONFIG_FILE}
+    echo "Finished converting control plane on node $n"
+  done
+
+}
+
+usage()
+{
+    echo "usage: kind_dual_conversion.sh [-n|--name <cluster_name>] [-ss secondary_service_subnet] [-sc secondary_cluster_subnet]"
+    echo "Convert a single stack cluster in a dual stack cluster"
+}
+
+parse_args()
+{
+    while [ "$1" != "" ]; do
+        case $1 in
+            -n | --name )			                              shift
+                                          	                CLUSTER_NAME=$1
+                                          	;;
+            -ss | --secondary-service-subnet )	           	shift
+                                          	                SECONDARY_SERVICE_SUBNET=$1
+                                          	;;
+            -sc | --secondary-cluster-subnet )	           	shift
+                                          	                SECONDARY_CLUSTER_SUBNET=$1
+                                          	                ;;
+            -h | --help )                       usage
+                                                exit
+                                                ;;
+            * )                                 usage
+                                                exit 1
+        esac
+        shift
+    done
+}
+
+parse_args $*
+
+# Set default values
+CLUSTER_NAME=${CLUSTER_NAME:-ovn}
+DUALSTACK_FEATURE_GATE="IPv6DualStack=true"
+SECONDARY_SERVICE_SUBNET=${SECONDARY_SERVICE_SUBNET:-"fd00:10:96::/112"}
+SECONDARY_CLUSTER_SUBNET=${SECONDARY_CLUSTER_SUBNET:-"fd00:10:244::/56"}
+
+# NOTE: ovn only
+export KUBECONFIG=${HOME}/admin.conf
+
+# KIND nodes
+NODES=$(kind get nodes --name ${CLUSTER_NAME})
+CONTROL_PLANE_NODES=$(kind get nodes --name ${CLUSTER_NAME} | grep control)
+WORKER_NODES=$(kind get nodes --name ${CLUSTER_NAME} | grep worker)
+
+# Create smoke deployment
```

Comment:
what is a smoke deployment?

Reply:
the idea is to deploy some application, i.e. a deployment with 2 http server pods, expose them with a service and check that it keeps working

Reply:
ok. maybe put that brief snippet as the comment, unless "smoke" is a common term for this kind of thing I haven't learned that yet.